.github/CHANGELOG.md

```diff
@@ -181,6 +181,7 @@
 1. [MCDU] Fixed options page text overlap - @MisterChocker (Leon)
 1. [CDU] Reworked Flight Phase system - @MisterChocker (Leon)
 1. [MISC] Fixed button labels not showing when unlit - @ImenesFBW (Imenes)
+1. [EXT VIEW] Adds cockpit.cfg and fixes flaps full angle in external view - @donstim (donbikes#4084)
```

Comment:
IMO I would put this as `MISC` instead of `EXT VIEW`
```suggestion
1. [MISC] Add cockpit.cfg and fixes flaps full angle in external view - @donstim (donbikes#4084)
```